### PR TITLE
Add option for PyCharm 2017.3.

### PR DESCRIPTION
**Pull request by ismailsunni**

### CODE REVIEW

File: deployment/ansible/development/site.yml [hunk 1]
```diff
@@ -18,5 +18,6 @@
         2016.3
         2017.1
         2017.2
+        2017.3
```
Comment (lucernae) [inline deployment/ansible/development/site.yml]: maybe you should make 2017.3 as the new default if you already tested it?

**Review by lucernae — COMMENTED** [review]
LGTM!
Does it work in the new one? I had some scripts that injects ssh pasword in the interpreter. I hope it works too.

**Comment by Gustry** [comment]
I was already using 2017.3 without this patch, it seems to be ok but I didn't try a lot

**Comment by ismailsunni** [comment]
It seems ok in my side.

> I was already using 2017.3 without this patch, it seems to be ok but I didn't try a lot

Perhaps you use 2017.2 first, then import the setting to 2017.3 when you installed 2017.3

**Pull request merged**